website/django/data/migrations/0003_data.py

```diff
@@ -18,11 +18,26 @@ def load_from_csv(apps, schema_editor):
         header = reader.next()
 
         for row in reader:
-
             # split Source column into booleans
             row_dict = dict(zip(header, row))
             for source in row_dict['Source'].split(','):
                 row_dict['Variant_in_' + source] = True
+            x = y = z = w = u = None
```

Comment:
Need comments to say what we're doing.

Put a big comment flag in about changing the 1 position.